plugins/mcreator-localization/help/default/particle/scale.md

```diff
@@ -0,0 +1 @@
+This parameter defines how many times visual particle size is larger than its physical size.
```

Comment:
Scale controls how much the particle texture should be scaled.

Reply:
:heavy_check_mark: Fixed